demo-client/src/main/java/com/illia/client/service/query/processor/unit/CountOperationProcessorUnit.java

```diff
@@ -0,0 +1,45 @@
+package com.illia.client.service.query.processor.unit;
+
+import com.illia.client.model.IMDbMovieEntity;
+import com.illia.client.model.request.entity.CountQueryEntity;
+import com.illia.client.model.request.entity.QueryEntity;
+import com.illia.client.model.request.registry.AttributeRegistry;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
+public class CountOperationProcessorUnit implements OperationProcessor {
+
+  @Override
+  public List<?> process(List<IMDbMovieEntity> records, QueryEntity queryEntity) {
+    CountQueryEntity countQueryEntity = (CountQueryEntity) queryEntity;
+    var attribute = countQueryEntity.getAttribute();
+
+    assert canGroupBy(attribute) : "Can't perform operation with this attribute!";
+
+    return records.stream()
+        .collect(Collectors.groupingBy(x -> x.getFieldAccessor(attribute)))
+        .entrySet()
+        .stream()
+        .map(x -> x.getKey() + ": " + x.getValue().size())
+        .sorted()
+        .collect(Collectors.toList());
+  }
+
+  private boolean canGroupBy(AttributeRegistry attribute) {
```

Comment:
It is more common to have a collection of these attributes and check with .contains(attribute)